apps/web/pages/home.tsx

```diff
@@ -0,0 +1,317 @@
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import TopBar from "../components/TopBar";
+import PlayButton from "../components/HomePage/PlayButton";
+import List from "../components/HomePage/List";
+import UserEntity from "../components/HomePage/UserEntity";
+import MatchEntity from "../components/HomePage/MatchEntity";
+import LeaderboardEntity from "../components/HomePage/LeaderboardEntity";
+import ArrayDoubleColumn from "../components/HomePage/ArrayDoubleColumn";
+import PlayMenu from "../components/HomePage/PlayMenu";
+import { setUserState } from "../store/UserSlice";
+import { useDispatch } from "react-redux";
+import { UserBack } from "../interface/UserInterface";
+import Link from "next/link";
+import ChatBar from "../components/chatBar";
+import playButtonStyles from "styles/playButton.module.scss";
+import textStyles from "styles/text.module.scss";
+import styles from "styles/home.module.scss";
+
+//temporary before the login page
+const user_id = "1edffd5a-863d-442c-a962-a5dbd9b2c686";
```

Comment:
This is not dynamic

Reply:
for me when the auth is ok you link the home page with the user_id

Reply:
That's because it should be a prop, probably with value string|undefined, if undefined redirect to login page, if defined display home.